server/api/routes/service.ts

```diff
@@ -0,0 +1,43 @@
+import * as mongoose from "mongoose";
+import * as jwt from "jsonwebtoken";
+import { UserModel } from "../models/user";
+import { HeaderMiddleware } from "../middlewares/header";
+import { ServiceController } from "../controllers/service";
+import { Request, Response, NextFunction } from "express";
+
+const User = mongoose.model("User", UserModel);
+
+export class ServiceRoutes {
+  public service: ServiceController = new ServiceController();
+
+  public routes(app): void {
+    app.use((req: Request, res: Response, next: NextFunction) => {
```

Comment:
Lets create a middleware for this because jwt verification would be used in other routes too.

Reply:
I will create a middleware for this in other PR, resolve it now